test/dsp.jl

```diff
@@ -21,6 +21,15 @@ a = [1.0,-2.4744161749781606,2.8110063119115782,-1.703772240915465,0.54443269488
 si = [0.9967207836936347,-1.4940914728163142,1.2841226760316475,-0.4524417279474106,0.07559488540931815]
 @test_approx_eq filt(b, a, ones(10), si) ones(10) # Shouldn't affect DC offset
 
+@test xcorr([1, 2], [3, 4]) == [4, 11, 6]
+
+@test fftshift([1 2 3]) == [3 1 2]
+@test fftshift([1, 2, 3]) == [3, 1, 2]
+@test fftshift([1 2 3; 4 5 6]) == [6 4 5; 3 1 2]
```

Comment:
probably should also test `fftshift(A, n)` for the 2d case.  And test that it throws an error if `n < 1` or `n < ndims(A)`.